tests/Aql/QueryHelper.cpp

```diff
@@ -45,16 +45,18 @@ auto vpackHasNoneRecursive(VPackSlice slice) -> bool {
   }
 
   if (slice.isArray()) {
-    auto iter = VPackArrayIterator(slice);
-    return std::any_of(iter.begin(), iter.end(),
-                       [](auto slice) { return vpackHasNoneRecursive(slice); });
-  }
-  if (slice.isObject()) {
-    auto iter = VPackObjectIterator(slice);
-    return std::any_of(iter.begin(), iter.end(), [](auto pair) {
-      return vpackHasNoneRecursive(pair.key) ||
-             vpackHasNoneRecursive(pair.value);
-    });
```

Comment:
It took me this long to figure out why you rewrite all these stl algorithms, because `end` now returns a sentinel, right?

Reply:
You could instead use `std::ranges::any_of(iter, [](auto pair) { ... });`. These are updated versions to support a different end-iterator type (in this case the default_sentinel)

Reply:
std::ranges not supported in clang + libstdc++ configuration.

I think it's common for our development proccess (sanitizers or local runs)

Reply:
But it can be boost...

Reply:
https://godbolt.org/z/WMhzrceWf proof

Reply:
https://godbolt.org/z/ec6vTKxjr proof

But any_of probably here probably can work

If I correct no problem with algorithm header but have with ranges